modules/002-deckhouse/templates/deckhouse-leader-service.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: deckhouse-leader
+  namespace: d8-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "deckhouse")) | nindent 2 }}
+spec:
```

```suggestion
spec:
  publishNotReadyAddresses: true
```
sorry, my bad, I forgot to mention this parameter at the beginning

---

I added the parameter